indico/modules/events/management/templates/clone_event.html

```diff
@@ -0,0 +1,107 @@
+{% from 'forms/_form.html' import form_header, form_fieldset, form_footer, form_row, form_rows %}
+{% from 'message_box.html' import message_box %}
+
+{% macro invisible_field(field) -%}
+    {% set field_value = field._value() if field._value else field.data %}
+    {% set data = [field_value] if field_value is string else ([field_value|tojson] if field_value is mapping else field_value) %}
+    {% for value in data %}
+        <input type="hidden" name="{{ field.name }}" value="{{ value | forceescape }}">
+    {% endfor %}
+{%- endmacro %}
+
+{% macro invisible_fields(form, skip) %}
+    {% for field in form %}
```

Comment:
There is no good reason to write code that can easily break if it can be done properly ;) `field.id` is the HTML ID which is just coincidentally identical to the name in most cases